blender-mesh/src/combine_indices/mod.rs

```diff
@@ -412,6 +423,7 @@ impl DerefMut for EncounteredIndexCombinations {
 
```

Comment:
Can we add a test in the tests module at the bottom that confirms that expanding material indices works properly for a mesh with more than one material